modules/rest-api/pages/rest-xdcr-adv-settings.adoc

```diff
@@ -320,68 +382,118 @@ It cannot be set when an individual replication is created, or when its settings
 | Default: 4.
 The maximum number of threads used per node, to support XDCR.
 A greater number of threads increases parallelism, and may thereby produce enhanced XDCR performance.
+
 This setting is only returned when global settings are retrieved; and can only be set when global settings are set.
 It cannot be set when an individual replication is created, or when its settings are modified.
 
+| `hlvPruningWindowSec`
+| Integer
+| Used in conflict-resolution. Developer Preview only.
```

Comment:
Used in *custom* conflict-resolution only

Reply:
Each Custom Conflict resolution document utilizes Hybrid-Logical Vectors (HLV), which is a specific type of version-vector that contains information about other participating clusters that were involved in modifying this document. Pruning window is defined as a time window that begins when a remote cluster has modified this document. If the pruning window is passed, and the cluster has not modified this document since, then the information about this cluster will be removed from the HLV. 

(Once custom conflict document is official, provide link here... concept will be too complicated to explain here)

Reply:
I have written: Used in _custom conflict-resolution_, and available in Developer Preview Mode only.
The integer specifies the duration, in seconds, of the time-window that starts when a remote cluster has modified the document that is currently being replicated.
The remote cluster's update is recorded in a _version-vector_.
If the specified time-window expires without further modification being made by the remote cluster, information about the remote cluster's update is removed from the version-vector.